spec/support/features/pages/schools/school_register_programme_wizard.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+require_relative "../base_page"
+
+module Pages
+  class SchoolRegisterProgrammeWizard < ::Pages::BasePage
```

Comment:
This class had to be added because we have two UI wizards for setting up Defaults for Academic Years which ask the same questions but in different orders.